AutolayoutPractice/RemindersList/Views/AddReminderViewController.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  AddReminderViewController.swift
+//  AutolayoutPractice
+//
+//  Created by Amanda Benavente on 6/23/24.
+//
+
+import UIKit
+
+class AddReminderViewController: UIViewController {
+
+    private let navBar = UINavigationBar()
```

Comment:
Embed in NavigationController to get default  navBar OR autolayout 2 buttons ** preferred